src/components/Mailbox.vue

```diff
@@ -94,7 +94,7 @@ export default {
 		},
 		initialPageSize: {
 			type: Number,
-			default: 20,
+			default: 30,
```

Comment:
yes but why do we need this for the horizontal split? horizontal split is the view with the fewest visible items in the list

Reply:
i can change the initialPageSize to be dynamic

Reply:
but why?

Reply:
so i can show different messages for different layouts.